### PR TITLE
[UBS courier] Improvement to 'Phone number' field validation #2840

### PR DESCRIPTION
**Pull request by BobNazar**
- Added  library google-libphonenumber for validating the 'Phone number' field

### CODE REVIEW

File: src/app/ubs/ubs/components/ubs-personal-information/ubs-personal-information.component.ts [hunk 1]
```diff
@@ -80,6 +81,7 @@ export class UBSPersonalInformationComponent extends FormBaseComponent implement
       this.personalDataForm.controls.address.setValue(data);
       this.personalDataForm.controls.addressComment.setValue(data.addressComment);
     });
+    console.log(this.personalData);
```
Comment (NewArtMask) [inline src/app/ubs/ubs/components/ubs-personal-information/ubs-personal-information.component.ts:84]: Delete console.log, please.

File: src/app/shared/phone-validator/phone.validator.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { AbstractControl, ValidatorFn } from '@angular/forms';
+import { PhoneNumberUtil, PhoneNumber } from 'google-libphonenumber';
+
+const phoneNumberUtil = PhoneNumberUtil.getInstance();
+
+export function PhoneNumberValidator(regionCode: string = undefined): ValidatorFn {
```
Comment (NewArtMask) [inline src/app/shared/phone-validator/phone.validator.ts:6]: We don't need undefined for initialization.

File: src/app/ubs/ubs/components/ubs-personal-information/ubs-personal-information.component.ts (hunk 1 above)
Comment (BobNazar) [reply src/app/ubs/ubs/components/ubs-personal-information/ubs-personal-information.component.ts]: done

File: src/app/shared/phone-validator/phone.validator.ts (hunk 2 above)
Comment (BobNazar) [reply src/app/shared/phone-validator/phone.validator.ts]: fixed

**Review by hnativlyubomyr — APPROVED** [review]

**Review by TarasKuzma1999 — APPROVED** [review]

**Review by SvitlanaVorobets — APPROVED** [review]

**Review by s1auka — APPROVED** [review]

**Review by NewArtMask — APPROVED** [review]

**Pull request merged**